lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -283,7 +283,8 @@ defmodule ExUnit.DocTest do
         error ->
           original_stack = System.stacktrace
           reraise ExUnit.AssertionError,
-            [message: "Doctest failed: got #{inspect(error.__struct__)} with message " <> inspect(Exception.message(error)),
+            [message: "Doctest failed: got #{inspect(error.__struct__)} with message " <>
+                      inspect(Exception.message(error)),
```

Comment:
Unnecessary change IMO, we don't have a line length standard in Elixir and while I agree these lines are quite long, these changes tend to make the PR less focused.